crates/ruff_linter/src/rules/flake8_logging_format/rules/logging_call.rs

```diff
@@ -55,8 +56,65 @@ fn check_msg(checker: &Checker, msg: &Expr) {
             _ => {}
         },
         // Check for f-strings.
-        Expr::FString(_) => {
-            checker.report_diagnostic_if_enabled(LoggingFString, msg.range());
+        Expr::FString(f_string) => {
+            if checker.is_rule_enabled(Rule::LoggingFString) {
+                let mut format_string = String::new();
+                let mut args = Vec::new();
+                let mut simple = true;
+
+                for f in f_string.value.f_strings() {
+                    for element in &f.elements {
+                        match element {
+                            InterpolatedStringElement::Literal(lit) => {
+                                format_string.push_str(lit.value.as_ref());
+                            }
+
+                            InterpolatedStringElement::Interpolation(interpolated) => {
+                                if interpolated.format_spec.is_some()
+                                    || !matches!(
+                                        interpolated.conversion,
+                                        ruff_python_ast::ConversionFlag::None
+                                    )
+                                {
+                                    simple = false;
+                                    break;
+                                }
+
+                                match interpolated.expression.as_ref() {
+                                    Expr::Name(name) => {
+                                        format_string.push_str("%s");
+                                        args.push(name.id.to_string());
+                                    }
+
+                                    _ => {
+                                        simple = false;
+                                        break;
+                                    }
+                                }
+                            }
+                        }
+                    }
+                }
+
+                if simple && !args.is_empty() {
+                    // Always emit comma-separated arguments, not a tuple, even for >1 arg
+                    let replacement = format!(
+                        "\"{}\"{}{}",
+                        format_string,
+                        if args.is_empty() {
+                            String::new()
+                        } else {
+                            String::from(", ")
+                        },
+                        args.join(", ")
+                    );
```

Comment:
nit: We usually try to use named arguments to `format!`. Something like this maybe:


```suggestion
                    let replacement = format!(
                        "\"{format_string}\"{sep}{args}",
                        sep = if args.is_empty() {
                            ""
                        } else {
                            ", "
                        },
                        args = args.join(", ")
                    );
```

I think you can also use string literals instead of converting to `String`s.